pkgs/servers/jicofo/default.nix

```diff
@@ -1,11 +1,11 @@
-{ pkgs, lib, stdenv, fetchurl, dpkg, jre_headless, nixosTests }:
+{ lib, stdenv, fetchurl, dpkg, jre_headless, nixosTests }:
 
 let
   pname = "jicofo";
-  version = "1.0-612";
+  version = "1.0-692-hf";
```

Comment:
Curious, does the hf mean anything?

Reply:
I wondered about this too. I noticed the following things:

- Since the first "-hf" package, there were no more non-"-hf" packages
- The "-hf" packages are in the same "stable" directory

While looking at this again I noticed there are actually releases in the GitHub repositories, so I will change to the latest release (which is synchronized across repositories) instead of the latest package on their server.